snaps/get-started/quickstart.mdx

```diff
@@ -19,18 +19,14 @@ See the following video demo:
 ## Prerequisites
 
 - [MetaMask Flask installed](install-flask.md)
-
-  :::tip
-  MetaMask Snaps works with the latest LTS version of Node.js, but we recommend using the version specified in
-  the template's [`.nvmrc`](https://github.com/MetaMask/template-snap-monorepo/blob/main/.nvmrc) file.
-  :::
-
 - A text editor (for example, [VS Code](https://code.visualstudio.com/))
+- [Node](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 18.16 or later
 - [Yarn](https://yarnpkg.com/) version 3
 
 ## Create the project
 
-Create a new Snap project using the [`@metamask/create-snap` CLI](https://github.com/MetaMask/snaps/tree/main/packages/create-snap) by running:
+Create a new Snap project using the [`@metamask/create-snap`](https://github.com/MetaMask/snaps/tree/main/packages/create-snap)
+CLI by running:
```

Comment:
Technically both the old text and this new text are incorrect. The CLI you are using in the below examples is yarn or npm. https://classic.yarnpkg.com/lang/en/docs/cli/create/

Reply:
Also, the README for our starter kit gives 3 examples: 

```
yarn create @metamask/snap your-snap-name
# or...
npx @metamask/create-snap your-snap-name
# or with npm v6 and later
npm create @metamask/snap your-snap-name
```

Reply:
Anyway, a better way to say it would be: 

```Create a new Snap project using the [`@metamask/create-snap`](https://github.com/MetaMask/snaps/tree/main/packages/create-snap) starter kit by running: